go.mod

```diff
@@ -344,6 +345,5 @@ replace (
 	sigs.k8s.io/cluster-api-provider-openstack => github.com/openshift/cluster-api-provider-openstack v0.0.0-20210820223719-a7442bb18bce
 	sigs.k8s.io/kustomize/kyaml => sigs.k8s.io/kustomize/kyaml v0.13.0
 	sigs.k8s.io/structured-merge-diff => sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06
-	sourcegraph.com/sourcegraph/go-diff => github.com/sourcegraph/go-diff v0.5.1
```

Comment:
why are these removed?

Reply:
I have no idea, I ran go mod vendor cause of the new packages I added. I'll do some digging to figure out why this has gone missing

Reply:
Could not find a good reason these were removed. Added them back in, ran go mod vendor and go mod tidy and committed the changes